.rubocop.yml

```diff
@@ -1,32 +1,73 @@
 AllCops:
+  Exclude:
+    - 'db/schema.rb'
+    - 'db/seeds.rb'
   TargetRubyVersion: 2.3
   TargetRailsVersion: 4.2
 
+Rails:
+  Enabled: true
+  
 Metrics/LineLength:
   Max: 100
 
 Metrics/BlockLength:
   Enabled: false
 
+Metrics/ModuleLength:
+  Exclude:
+    - 'spec/**/*.rb'
+
+Metrics/BlockLength:
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*.rb'
+
+Metrics/MethodLength:
+  Exclude:
+    - 'spec/**/*.rb'
+
+Metrics/AbcSize:
+  Exclude:
+    - 'spec/**/*.rb'
+
+Metrics/CyclomaticComplexity:
```

Comment:
whoa! i'd love to know what cyclomatic complexity is!

Reply:
"the number of linearly independent paths through a method"

Reply:
detail: https://rubocop.readthedocs.io/en/latest/cops_metrics/#metricscyclomaticcomplexity